WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -467,15 +527,29 @@ public void onSuccess(long personID, int localTableBlogId) {
 
             @Override
             public void onError() {
+                int errorMessageRes;
+                switch (personType) {
+                    case USER:
+                        errorMessageRes = R.string.error_remove_user;
+                        break;
+                    case VIEWER:
+                        errorMessageRes = R.string.error_remove_viewer;
+                        break;
+                    default:
+                        errorMessageRes = R.string.error_remove_follower;
```

Comment:
Only remaning cases are the follower ones. Adding an exhaustive list of enums seems to create a variable might not be initialized error. So, I did it this way to avoid an extra initilization check.